server/dash/dashboards/neighborhood.py

```diff
@@ -39,27 +42,80 @@ def populate_options():
         )
     return values
 
+print(populate_options())
+
+layout = html.Div([
+    html.H1(title),
+    dcc.Dropdown(
+        id='council_list',
+        clearable=False,
+        value="Arleta",
+        placeholder="Select a neighborhood",
+        options=populate_options()
+    ),
+    dcc.Dropdown(
+        id='data_type',
+        clearable=False,
+        placeholder="Select a timeframe",
+        options=[
+            {'label': 'Compare to: Daily Council Average', 'value': 'nc_avg'},
+            {'label': 'Compare to: Weekly Council Average', 'value': 'nc_avg_wk'},
+            {'label': 'Compare to: Monthly Council Average', 'value': 'nc_avg_Mo'},
+        ],
+
+        value='nc_avg'
+    ),
+    dcc.Graph(
+        id='graph1',
+        figure=fig,
+        config=CONFIG_OPTIONS
+    ),
+
+    dcc.Graph(
+        id='graph2',
+        figure=fig,
+        config=CONFIG_OPTIONS
+    )
+])
+
 
 # Define callback to update graph
 @app.callback(
     Output('graph1', 'figure'),
-    [Input("council_list", "value")]
+    [Input("council_list", "value"),
+    Input("data_type", "value")]
 )
-def update_figure(selected_council):
-
+def update_figure(selected_council, select_timeframe):
     neighborhood_sum_df = df[df.council_name == selected_council].groupby(['created_date']).agg('sum').reset_index()  # noqa
+
     total_sum_df = df.groupby(['created_date']).agg('sum').reset_index()
+
     total_sum_df["nc_avg"] = total_sum_df["counts"] / 99
-    merged_df = neighborhood_sum_df.merge(total_sum_df["nc_avg"].to_frame(), left_index=True, right_index=True)  # noqa
 
+    total_sum_df["week_n"] = total_sum_df['created_date'].apply(lambda x: str(x.isocalendar()[1]))
+    total_sum_df["month_n"] = total_sum_df['created_date'].dt.month
+
+    total_sum_df['nc_avg_Mo'] = total_sum_df.groupby(['month_n'])['counts'].transform('mean')
```

Comment:
nit: let's not mix in uppercase here. 'nc_avg_mo'

Reply:
Thanks. Do we have instructions on best coding practices and naming conventions for this project?

Reply:
I don't think any style guide has been followed so far, but the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html) is preferred.

I'm not sure if this particular style convention would be mentioned in that guide above, but in general, Python sticks to `snake_case` for variable names (and I think dict keys would fall under that) and `CamelCase` for class names. Mixing `snake_case` and `CamelCase` (e.g., `nc_avg_Mo`) is not recommended.

Reply:
Thanks